database/migrations/2023_12_19_211314_user_unique_email.php

```diff
@@ -4,12 +4,21 @@
 use Illuminate\Database\Schema\Blueprint;
 use Illuminate\Support\Facades\Schema;
 
+require_once 'include/common.php';
```

Comment:
If there's external code you want to use (excluding Laravel), you should copy that code into the migration to ensure that the migration continues to work even after we delete/modify the functions being imported.